modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -34,39 +40,47 @@
 
 import javax.servlet.http.HttpServletRequest;
 
+import org.hamcrest.CoreMatchers;
+
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 import org.mockito.Matchers;
 import org.mockito.Mock;
+import org.mockito.Mockito;
 
 import org.powermock.api.mockito.PowerMockito;
 import org.powermock.api.mockito.expectation.PowerMockitoStubber;
 import org.powermock.api.support.membermodification.MemberMatcher;
+import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 
 import org.springframework.mock.web.MockHttpServletRequest;
 
 /**
  * @author Pedro Queiroz
  */
+@PrepareForTest(RequestBackedPortletURLFactoryUtil.class)
 @RunWith(PowerMockRunner.class)
 public class DocumentLibraryDDMFormFieldTemplateContextContributorTest
 	extends BaseDDMFormFieldTypeSettingsTestCase {
 
 	public HttpServletRequest createHttpServletRequest() {
-		MockHttpServletRequest request = new MockHttpServletRequest();
+		MockHttpServletRequest httpServletRequest =
+			new MockHttpServletRequest();
 
 		ThemeDisplay themeDisplay = new ThemeDisplay();
 
 		themeDisplay.setPathContext("/my/path/context/");
 		themeDisplay.setPathThemeImages("/my/theme/images/");
 
-		request.setAttribute(WebKeys.THEME_DISPLAY, themeDisplay);
+		httpServletRequest.setAttribute(WebKeys.THEME_DISPLAY, themeDisplay);
+
```

Comment:
@carolmariaabb, I believe this blank line isn't necessary, please check the lines 76 and 77 above.

Reply:
Hi @natocesarrego. When I execute the SF command, the following error is displayed "`There should be an empty line before line '80', as we finished referencing variable 'themeDisplay`'". So I believe this blank line is necessary 😅

Reply:
Who am I to disagree with SF?! 😁 

Thanks for the explanation, @carolmariaabb.